ipfscluster.go

```diff
@@ -53,6 +53,15 @@ type Consensus interface {
 	Clean(context.Context) error
 	// Peers returns the peerset participating in the Consensus
 	Peers(context.Context) ([]peer.ID, error)
+	// IsTrustedPeer returns true if the given peer is "trusted"
+	// This will grant access to more rpc endpoints and a
+	// non-trusted one. This should be fast as it will be
+	// called repeteadly for every remote RPC request.
+	IsTrustedPeer(context.Context, peer.ID) bool
+	// Trust marks a peer as "trusted"
```

Comment:
// Trust marks a peer as "trusted".
// This will grant access to more rpc endpoints than a
// non-trusted one. This should be fast as it will be
// called repeatedly for every remote RPC request.